src/views/Graph.vue

```diff
@@ -526,6 +559,27 @@ export default {
           ]
         `)
       }
+
+      if (this.groupCyclePoint) {
+        // Loop over the subgraphs
+        Object.keys(cycleSubgraph).forEach((key, i) => {
+          // Loop over the nodes that are included in the subraph
+          const nodeFormattedArray = cycleSubgraph[key].map(a => `"${a.id}"`)
+          ret.push(`
+          subgraph cluster_margin
+          {
+            margin=50.0
+            label="margin"
```

Comment:
Very neat margin trick! I did not know we could do this!

But I don't think it's doing quite what it says on the tin, because all of the `cluster_margin` subgraphs have been given the same name, so all of the `cluster_<i>` subgraphs are residing in the same margin. It does seem to work, if I increase the margin the clusters move apart, though not by the amount I would expect, but I really don't understand how this works.

If we give each `cluster_margin` subgraph a unique ID, then the margin applies to each contained subgraph individually not globally.

```suggestion
          subgraph cluster_margin_$(i)
          {
            margin=50.0
            label="margin"
```

Might be worth increasing this margin value, I had a play with the "complex" workflow which has a nasty graph and found ~100 worked well (I can get you a copy of the complex workflow, good for testing nasty use cases).